Ops/WebOperations.cs

```diff
@@ -47,7 +45,7 @@ public static string AuthenticateAgent(string AgentName, string ServerName)
             }
             catch (Exception ex)
             {
-                throw ex;
+                throw new Exception("Authentication Fail", ex);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['System.Exception' should not be thrown by user code.](https://app.codacy.com/app/onurakkaya/DSM.Core/pullRequest?prid=4732054)